sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -9,11 +9,11 @@
 #include "azure/core/nullable.hpp"
 #include "azure/core/response.hpp"
 #include "azure/storage/common/crypt.hpp"
-#include "azure/storage/common/json.hpp"
 #include "azure/storage/common/storage_common.hpp"
 #include "azure/storage/common/storage_exception.hpp"
 #include "azure/storage/common/xml_wrapper.hpp"
 
+#include <nlohmann/json.hpp>
```

Comment:
Headers files should not include JSON library. Include JSON from each cpp file that uses it.